env-defaults

```diff
@@ -82,6 +82,12 @@ DEBUG_KNEX=false
 # Log level (error, warn, info, debug)
 LOG_LEVEL=debug
 
+# Default number of results in a page
+DEFAULT_PAGE_SIZE=2000
```

Comment:
We're changing our default page size from 10 to 2000 and max page size from 2000 to 10000?

I think default should stay at 10 for performance and usability reasons. I don't have as strong of an opinion on the change from 2000 to 10000 max.

Reply:
I'm just following the AC...